db/blip_handler.go

```diff
@@ -343,7 +343,7 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 	_, forceClose := generateBlipSyncChanges(changesDb, channelSet, options, opts.docIDs, func(changes []*ChangeEntry) error {
 		base.DebugfCtx(bh.loggingCtx, base.KeySync, "    Sending %d changes", len(changes))
 		for _, change := range changes {
-			if !strings.HasPrefix(change.ID, "_") {
+			if !strings.HasPrefix(change.ID, "_") { //nolint: nestif
```

Comment:
Do you think `nestif` is overly sensitive with the default settings? We could consider tweaking or disabling that linter entirely.

It seems to have a problem with a lot of code that doesn't warrant being flagged IMO, and would reduce the `//nolint` noise.

Reply:
We could up the amount of cyclomatic complexity for this but I think generally anywhere it finds nestif problems it _probably_ should be a separate function. This is a personal bias.

Reply:
Since this doesn't seem configurable in .golangci.yml I'm just going to skip nestif linter. We already disable `gocyclo` which does similar things and it's visibly obvious when a function is too big/complicated and should be spotted in a code review.

Reply:
Yeah, I agree that it's obvious during review if things need to be split more. Don't need to rely on the linter for this.